source/configuration_manual/sieve/extensions/enotify.rst

```diff
@@ -0,0 +1,78 @@
+.. _pigeonhole_extension_enotify:
+
+=============================================
+Pigeonhole Sieve: Extension for Notifications
+=============================================
+
+
+The Sieve **enotify** extension
+(`RFC5435 <http://tools.ietf.org/html/rfc5435/>`__) adds notify action to the Sieve language.
```

Comment:
+the there somewhere

Reply:
what?

Reply:
adds *the*/*a* notify action to the Sieve language.